ppml/trusted-big-data-ml/python/docker-graphene/kubernetes/bigdl-ppml-helm/values.yaml

```diff
@@ -2,3 +2,16 @@ dataPath: path_to_your_data_on_host
 image: your_ppml_image
 k8sMaster: k8s://https://master_ip:master_port
 pvc: your_pvc
+
```

Comment:
please order the fields and also rename them as:
deployMode:
driverCores
driverMem
executorCores
executorMem
executorInstances
sgx
sgxLog
sgxDriverMem
sgxDriverJvmMem
sgxExecutorMem
sgxExecutorJvmMem